browser-based-ingestion/src/getProcessedVideo.js

```diff
@@ -0,0 +1,87 @@
+import { putMedia } from './AWS-SDKCalls.js'
+// eslint-disable-next-line no-undef
+const { createFFmpeg } = FFmpeg
```

Comment:
I would get `FFmpeg` via webpackage/npm so you don't have to do `no-undef`

Reply:
I have removed dependency on node and webpack recently. I referred to the example for using FFmpeg with browsers, and this is how they have used it https://github.com/ffmpegwasm/ffmpeg.wasm/blob/master/examples/browser/transcode.html#L23

Reply:
From the link, I can see that they're trying to make the example simple. It's okay to have node and webpack. It's very common to use them for the precompilation part, where as your final product will still be pure static files that you can serve to browsers.

Another benefit of precompiling your source codes is to make your codes backward compatible. From what I see from your codes, you're using some new features, such as async/await, const, es imports, etc. These might run fine on newest browsers, but in older browsers they have older javascript runtimes that don't know how to parse these syntaxes. So, normally people would transcompile their javascript codes with [Babel](https://babeljs.io/), to translate these new syntaxes to workaround older syntaxes.

I also personally prefer getting packages from npm, this makes it easier for people to integrate their larger projects.